Update ne-d3d12-d3d12_logic_op.md

walbourn opened the pull request:
This is a lot easier to read as pseudo-code than trying to reconstruct the bitwise logic from the English text.

walbourn committed:
Update ne-d3d12-d3d12_logic_op.md

This is a lot easier to read as pseudo-code than trying to reconstruct the bitwise logic from the English text.

PRMerger13 commented:
@walbourn : Thanks for your contribution! The author(s) have been notified to review your proposed change.

stevewhims committed:
Just a single backtick for inline code

PRMerger15 commented:
@stevewhims : Thanks for your contribution! The author(s) have been notified to review your proposed change.

stevewhims committed:
Working around build system bug

So, I changed two backticks to one for inline code, which is valid markdown. But an issue with our build system means that we have to use pure HTML *in the descriptions of Win32 enums*. So I replaced the markdown with HTML.

PRMerger17 commented:
@stevewhims : Thanks for your contribution! The author(s) have been notified to review your proposed change.
